data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/OpenSearchSink.java

```diff
@@ -249,11 +268,8 @@ private void doInitializeInternal() throws IOException {
     }
 
     final int maxRetries = openSearchSinkConfig.getRetryConfiguration().getMaxRetries();
-    final OpenSearchClient filteringOpenSearchClient = openSearchClient.withTransportOptions(
-            TransportOptions.builder()
-                    .setParameter("filter_path", "errors,took,items.*.error,items.*.status,items.*._index,items.*._id")
-                    .build());
```

Comment:
This filtering part is removed. Is it not needed any more?

Reply:
It is included in L222-229. But I should have missed it in the constructor of openSearchClientRefresher. Will make the change